src/d3d9/d3d9_device.cpp

```diff
@@ -5190,25 +5190,58 @@ namespace dxvk {
     // First we calculate the size of that UP buffer slice
     // and store all sizes and offsets into it.
 
+    struct VBOCopy {
+      uint32_t srcOffset;
+      uint32_t dstOffset;
+      uint32_t copyBufferLength;
+      uint32_t copyElementCount;
+      uint32_t copyElementSize;
+      uint32_t copyElementStride;
+    };
     uint32_t upBufferSize = 0;
```

Comment:
Usage of this variable implies it's a rolling value. Perhaps renaming it into `upBufferRollingSize` (or something comparable) would make this fact more clear. (Or perhaps there already is a naming convention in similar code that could be applied here too?)

Reply:
What's a rolling value?

`upBufferSize` just sums up the size of the vertex data for all vertex buffers and the index buffer so we can then allocate a single UpBuffer slice rather than needing multiple slices.

Reply:
Indeed. It starts at zero and is incremented in each for loop iteration. 
Of course, to you it is probably obvious this is the way this works, but this usage can be seen as a pattern that happens elsewhere as well, and capturing that fact via a naming convention would make this clearer to a reader/maintainer of the code base.
Maybe "rolling" is not the most accurate terminology, perhaps something like "incremental" is the better word to describe this usage, but that's longer.
Feel free to ignore this reasoning, it's just a suggestion.

Reply:
I could call it something like `totalUpBufferSize`. How about that?